lib/logstash/outputs/elasticsearch.rb

```diff
@@ -33,6 +33,24 @@
 # If using the default `protocol` setting ("node"), your firewalls might need
 # to permit port 9300 in *both* directions (from Logstash to Elasticsearch, and
 # Elasticsearch to Logstash)
+#
+# ## Retry Policy
+#
+# Not all events always make it successfully to Elasticsearch, and they are not always 
+# formatted correctly for the index they are targeting. So that we minimize loss of 
```

Comment:
Change to: "For example, there could be events  which are not formatted correctly for the index they are targeting (type mismatch in mapping). So that we minimize...."